src/variable/VRLadle.sol

```diff
@@ -297,26 +297,6 @@ contract VRLadle is UUPSUpgradeable, AccessControl() {
 
     // ---- Asset and debt management ----
 
-    /// @dev Move collateral and debt between vaults.
-    function stir(
```

Comment:
Why was `stir` moved exactly?

Reply:
It doesn't update the global debt counters when `ilkId != baseId`. It could be fixed, but at the same time we haven't ever used it, so better to just reduce the attack surface.